src/Orleans/Streams/Predicates/ExactMatchStreamNamespacePredicate.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Orleans.Streams
+{
+    [Serializable]
+    internal class ExactMatchStreamNamespacePredicate : IStreamNamespacePredicate
```

Comment:
why is ExactMatchStreamNamespacePredicate  internal? it should be public to external users right?